src/javascript/binary/websocket_pages/trade/tick_trade.js

```diff
@@ -314,36 +314,20 @@ WSTickDisplay.dispatch = function(data) {
       window.tick_init = 'initialized';
     }
   }
-  if (data.tick) {
-    spots2 = Tick.spots();
-    epoches = Object.keys(spots2).sort(function(a, b) {
-        return a - b;
-    });
-  } else if (data.history) {
-    epoches = data.history.times;
+  
+  if (data.history) {
+      ticks = Tick.history2ticks(data.history);
```

Comment:
@AlexShpak Is it needed to change the `var epoches, ...` to `var ticks, ...` as well?